src/data-access/repositories/mongo/mongo-ingested-entity-converter.ts

```diff
@@ -57,6 +57,7 @@ export interface MongoIngestedSegment extends MongoId {
   identifier?: string
   metaData?: unknown // This is the current spelling in the database from Core... TOD: Update when we control Ingest
   budgetDuration?: number
+  invalidity: { reason: string }
```

Comment:
I'm probably not considering something, so mostly for curiosity:
Why is this not optional on ingested entities?

Reply:
Because of oversight.
It is now optional :)

Reply:
Well that makes sense then! :-D